docs/scripts/preprocessor.py

```diff
@@ -46,10 +41,7 @@ def get_markdown_row_data(row):
     data
         ['apples', 'red', 'a']
     """
-    data = row.split("|")
-    data = data[0:len(data)-1]
-    data = [e.strip() for e in data if e.strip()]
-    return data
+    return [e.strip() for e in row.split("|")[1:-1] if e.strip()]
```

Comment:
I don't think this works for blank cells, does it? i.e. `|cat|dog||`

Reply:
fix checked in. you are right.